src/amb_kf.c

```diff
@@ -231,17 +354,18 @@ void nkf_update(nkf_t *kf, double *measurements)
               kf->obs_dim, kf->decor_mtx, kf->obs_dim, /*  N, A, lda. */
               resid_measurements, 1); /*  X, incX. */
 
-  /*  predict_forward(kf);. */
+  /*  Temporal update */
```

Comment:
Temporal update seems like it could potentially be confusing as its non standard terminology and in our case only refers to an update of the covariance.